WireUI/Sources/WireConversationUI/ConversationCell/CellTypes/TimeDivider/TimeDividerContentView.swift

```diff
@@ -0,0 +1,128 @@
+//
+// Wire
+// Copyright (C) 2025 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+import WireDesign
+
+struct TimeDividerContentView: ConversationCellContentViewProtocol {
+
+    private let dividerColor = ColorTheme.Strokes.outline.color
+
+    private(set) var model: TimeDividerModel
+
+    var body: some View {
+        HStack(spacing: 0) {
+
+            if model.isUnreadIndicatorVisible {
+                unreadDot
+            } else {
+                leadingDivider
+            }
+
+            if !model.text.isEmpty {
+                text
+            }
+
+            trailingDivider
+
+        }
+        .padding(.vertical, 8)
+    }
+
+    @ViewBuilder private var withoutUnreadIndicator: some View {
+
+        leadingDivider
+
+        if !model.text.isEmpty {
+            text
+        }
+
+        trailingDivider
+
+    }
+
+    @ViewBuilder private var text: some View {
+        Text(model.text)
+            .multilineTextAlignment(.center)
+            .font(.footnote)
+            .fontWeight(.semibold)
+            .padding(.horizontal, 12)
+            .layoutPriority(1)
+    }
+
+    @ViewBuilder private var unreadDot: some View {
+        Circle()
+            .fill(.tint)
```

Comment:
question: does this work with accentColor?

Reply:
Yes, works seamlessly.
<img width="445" alt="Screenshot 2025-03-21 at 18 13 50" src="https://github.com/user-attachments/assets/cbebd931-fb4f-4680-a061-cf9ce3f78e94" />
<img width="445" alt="Screenshot 2025-03-21 at 18 14 39" src="https://github.com/user-attachments/assets/20737310-35c6-4a61-80b0-32ed2759656b" />